org.eclipse.scout.rt.rest/src/main/java/org/eclipse/scout/rt/rest/ExtRestApplication.java

```diff
@@ -20,11 +20,11 @@ public class ExtRestApplication extends RestApplication {
   @Override
   protected boolean filterClass(Class<?> clazz) {
     return !AntiCsrfContainerFilter.class.isAssignableFrom(clazz)
-        && (!IRestResource.class.isAssignableFrom(clazz) || isExtScope(clazz));
+        && isExtScope(clazz);
   }
 
   protected boolean isExtScope(Class<?> clazz) {
     RestApplicationScope annotation = clazz.getAnnotation(RestApplicationScope.class);
-    return annotation != null && ObjectUtility.isOneOf(RestApplicationScopes.EXT, annotation.value());
+    return (annotation == null && !IRestResource.class.isAssignableFrom(clazz)) || (annotation != null && ObjectUtility.isOneOf(RestApplicationScopes.EXT, annotation.value()));
```

Comment:
this code snippet is a bit hard to read, suggestion:
```
  protected boolean isExtScope(Class<?> clazz) {
    RestApplicationScope annotation = clazz.getAnnotation(RestApplicationScope.class);
    if (annotation != null) {
      // check matching scope
      return ObjectUtility.isOneOf(RestApplicationScopes.EXT, annotation.value());
    }
    // include all non-IRestResource classes
    return !IRestResource.class.isAssignableFrom(clazz);
  }
```

Reply:
Reverted https://github.com/eclipse-scout/scout.rt/compare/6afdcd507e7cd4fef7c7ed815fdbd73c632c1996..a3cc68288dc0cb782cd10418693764f45cc3fd31

Reply:
Do not revert, change is necessary otherwise `AbstractApiExposedFeature` is always installed also for this application (even though it should be ignored).

Reply:
https://github.com/eclipse-scout/scout.rt/compare/a3cc68288dc0cb782cd10418693764f45cc3fd31..b38d059fe82f7363935dbdc83e8b4cf671275983